day_two_guide/host_level_tasks.adoc

```diff
@@ -63,6 +63,11 @@ for various node management options. These include:
 * xref:../admin_guide/manage_nodes.adoc#evacuating-pods-on-nodes[Evacuating Pods on Nodes]
 * xref:../admin_guide/managing_pods.adoc#managing-pods-poddisruptionbudget[Setting Pod Disruption Budgets]
 
+A node can reserve a portion of its resources to be used by specific components. These include the kubelet, kube-proxy, Docker, or other remaining system components such as sshd and NetworkManager. See the
+xref:../admin_guide/allocating_node_resources.adoc#admin-guide-allocating-node-resources[Allocating
```

Comment:
Do we need to point out that it's in a different book? Maybe s/See the xref[Allocating node resources section in the Cluster Administrator guide] for more information on how to configure a node's resources to be used by these components./You can xref[allocate the node's resources] to these components.

Reply:
@kalexand-rh There was a recent crop of BZs about people not seeing links inline of text, and that coupled with the knowledge that people still print out our docs has made me conscious of mentioning the location. It's not adding too much, I think. I've taken away some of the word, though.